website_sale_require_legal/views/assets.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2017 Jairo Llopis <jairo.llopis@tecnativa.com>
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <template id="assets_frontend" inherit_id="website.assets_frontend">
+        <xpath expr=".">
+            <script
+                type="text/javascript"
+                src="/website_sale_require_legal/static/src/js/tour.js"
```

Comment:
Move tour.js to a new location: /website_sale_require_legal/static/tests/tours/tour.js